bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/util/WebUtils.java

```diff
@@ -0,0 +1,20 @@
+package org.bahmni.module.bahmnicore.util;
+
+import org.openmrs.module.webservices.rest.SimpleObject;
+
+import java.util.LinkedHashMap;
+
+public class WebUtils {
+
+    public static SimpleObject wrapErrorResponse(String code, String reason) {
+        LinkedHashMap map = new LinkedHashMap();
+        if (reason != null && !"".equals(reason)) {
```

Comment:
[Minor] Maybe in future we could consider using String.isEmpty( ) method. Or String.trim().isEmpty().

Reply:
We still need to a null check. and since its only an internal util function, I think its ok .. could do something like...  !"".equals(reason.trim()) ... but I think its sufficient for now

Reply:
I agree.